core/src/code.rs

```diff
@@ -124,11 +124,6 @@ impl Code {
             return Err(CodeError::StartSectionExists);
         }
 
-        if module.custom_sections().count() != 0 {
-            log::debug!("Found custom sections in contract code, which is not allowed");
-            return Err(CodeError::CustomSectionsExist);
```

Comment:
The error variant should also be removed since it doesn't get used anywhere else?